bl-user-setup

```diff
@@ -18,15 +18,15 @@
 #    along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 # USER="$1" USER is exported by lightdm
```

Comment:
If USER is user@domain, does `getent passwd` show the the user as `user` or `user@domain`? I think it would be just user.